cmd/internal/dgather.go

```diff
@@ -361,20 +367,55 @@ func RenderJobBPM(currentJob manifest.Job, jobInstances []manifest.JobInstance,
 			}
 
 			// Parse a rendered bpm.yml into the bpm Config struct
-			jobInstances[i].BPM, err = bpm.NewConfig(bpmBytes)
+			jobIndexBPM[i], err = bpm.NewConfig(bpmBytes)
 			if err != nil {
 				return err
 			}
+			instanceBytes := generateJobInstanceSHA(currentJob, jobInstances[i])
+
+			// All instances SHA are the same, while all instances are based on index 0
+			jobInstances[i].Fingerprint = generateSHA(instanceBytes)
+		}
 
-			// Consider adding a Fingerprint to each job instance
-			// instance.Fingerprint = generateSHA(fingerPrintBytes)
+		for _, jobBPMInstance := range jobIndexBPM {
+			if !reflect.DeepEqual(jobBPMInstance, jobIndexBPM[0]) {
+				log.Infof("found different BPM job indexes for job %v in manifest %v, this is NOT SUPPORTED", currentJob.Name, manifestName)
```

Comment:
add a `WARNING: ` in front of it please.

Reply:
done